isobus/include/isobus/isobus/isobus_virtual_terminal_client.hpp

```diff
@@ -1370,6 +1370,8 @@ namespace isobus
 		/// @returns true if the message was sent
 		bool send_delete_object_pool() const;
 
+		bool send_working_set_maintenance(bool initializing) const;
```

Comment:
issue: there are now two send_working_set_maintenance functions defined in the header file. 

Try to modify the old one instead of adding a new one

Reply:
Should I just remove the one which takes VTVersion workingSetVersion as a parameter? I thought there would be no need for this one since you want the version to be 6 all the time.